IndexedDB/remove-listener-in-handler.any.js

```diff
@@ -0,0 +1,43 @@
+// META: script=support-promises.js
+
+const DB = databaseName("");
```

Comment:
nit: Both `DB` and `db` are used as variable names here... maybe call this `DB_NAME` ?